relentless/simulate/simulate.py

```diff
@@ -109,3 +112,288 @@ class Data:
     @abc.abstractmethod
     def __call__(self, sim):
         pass
+
+class Potentials:
+    """Combination of multiple potentials.
+
+    Iniitializes a :py:class:`PairPotentialTabulator` object that can store multiple potentials.
```

Comment:
Can you also tell the user that they will need to set `rmax` and `num` (I think that's the right names) for `pair` before the `Potentials` object can be used (if `pair is not None`)?